confirmation/models.py

```diff
@@ -100,8 +100,11 @@ def create_confirmation_link(
 
     current_time = timezone_now()
     expiry_date = None
-    if validity_in_days:
-        expiry_date = current_time + datetime.timedelta(days=validity_in_days)
+    if validity_in_days is not None:
+        if validity_in_days == -1:
```

Comment:
We currently use `-1` as API level value. We can also use a string like `never` but I feel using `-1` is better.

Reply:
Probably a good question for "API design".